src/index.html

```diff
@@ -3,11 +3,530 @@
+                  <li class="features__item">
+                    Feel-in-chest Base Power
+                  </li>
+
+                  <li class="features__item">
+                    Lossless Digital Audio Transmission
+                  </li>
+
+                  <li class="features__item">
+                    Easy & Stable Stereo Pairing
+                  </li>
+
+                  <li class="features__item">
+                    Crisp and Clear High Frequency Sound
+                  </li>
+
+                  <li class="features__item">
+                    Streams from Cloud Music and Local Library
+                  </li>
+
+                  <li class="features__item">
+                    Auto Music Playback from Last Song Stopped
+                  </li>
+                </ul>
+
+                <div class="features__img">
+                  <img
+                  src="./images/features-vector-photo.svg"
+                  alt="features vector photo"
+                  class="features__vector-photo"
+                >
+                </div>
+
+                <h3 class="features__second-title features__second-title--hidden">
+                  Connectivity
+                </h3>
+
+
+                <ul class="features__info-list features__info-list--hidden">
+                  <li class="features__item">
+                    Hands Free Wireless Audio
+                  </li>
+
+                  <li class="features__item">
+                    Bluetooth 4.0 LE
+                  </li>
+
+                  <li class="features__item">
+                    Wi-Fi 2.4 GHz (802.11 b/g/n)
+                  </li>
+
+                  <li class="features__item">
+                    Smart Multiroom System Set Up
+                  </li>
+
+                  <li class="features__item">
+                    Party Mode with 6.0 Units and above
+                  </li>
+
+                  <li class="features__item">
+                    MESHNET Multi Speaker Network
+                  </li>
+                </ul>
+
+                <h3 class="features__second-title features__second-title--hidden-right">
+                  App Features
+                </h3>
+
+                <ul class="features__info-list features__info-list--hidden-right">
+                  <li class="features__item">
+                    Customize Music Schedule
+                  </li>
+
+                  <li class="features__item">
+                    Wake Up with Favorite Songs
+                  </li>
+
+                  <li class="features__item">
+                    Home Detection Auto Wake Up
+                  </li>
+
+                  <li class="features__item">
+                    Color Wheel
+                  </li>
+                </ul>
+
+                <div class="features__bottom">
+                  <img
+                    src="./images/features-background.png"
+                    alt="features__background-img"
```

Comment:
The `alt` attribute contains information that will be displayed if the photo cannot be loaded, as well as voiced by the screen reader, so there should be a description of what the image contains.